### ISSUE TITLE
Error when running on OSX

### ISSUE THREAD

**Issue by shyam-habarakada** [snw35/cloudenv#6]
I installed this on a Mac and on first run, got this error (in docker container logs)

```
No matching internal group found, creating one...
groupadd: GID '20' already exists
```
It looks like the check to see if the group exists, in https://github.com/snw35/cloudenv/blob/master/docker-entrypoint.sh#L70 is not working as intended.

Looking into the image we have,

```bash
bash-5.1# cat /etc/group | grep 20
dialout:x:20:root
nofiles:x:200:
smmsp:x:209:smmsp
```

There is a group called `dialout` that has the id `20` assigned. That appears to be clashing with my current HOST_GROUP_ID which is also 20. The code in docker-entrypoint.sh to detect this scenario is not working as far as I can tell, because it is falling into the flow as if there was no conflict.

I was able to work around this issue by _hacking_ the `cloudenv` script (locally) to pass in an empty value into the `HOST_GROUP_ID` and that worked. However, I am not sure what the right fix is because it is unclear what the original use-case may have been for the _"use existing group if it matches"_ scenario. @snw35 any advice?

**Comment by snw35** [comment]
Thanks for reporting this,

I suspect the behaviour of the "id -g" command changed after I added the core-utils package to the container, e.g switched to the full version rather than the busybox cut-down one.

I've changed the check to use "getent group" instead which should fix this and be more reliable in general.

Let me know if the latest build fixes this for you (tag 2021-05-17 or just pulling and using 'latest').
